applications/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -534,6 +536,7 @@ static void set_conn_params(struct bt_conn *conn, bool peer_llpm_support)
 	int err;
 
 	if (peer_llpm_support && IS_ENABLED(CONFIG_BT_LL_NRFXLIB)) {
+#ifdef CONFIG_BT_LL_NRFXLIB
```

Comment:
if we use preprocessor `IS_ENABLED` is not needed

Reply:
fixed